src/opamPrinter.mli

```diff
@@ -69,11 +80,17 @@ val format_opamfile: Format.formatter -> opamfile -> unit
 *)
 module Normalise : sig
   val escape_string : string -> string
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Normalise.escape_string instead."]
   val value : value -> string
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Normalise.value instead."]
   val item : opamfile_item -> string
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Normalise.item instead."]
   val item_order : opamfile_item -> opamfile_item -> int
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Normalise.item_order instead."]
   val items : opamfile_item list -> string
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Normalise.items instead."]
   val opamfile : opamfile -> string
+  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Normalise.opamfile instead."]
 end
```

Comment:
```suggestion
module Normalise : sig
  val escape_string : string -> string
  val value : value -> string
  val item : opamfile_item -> string
  val item_order : opamfile_item -> opamfile_item -> int
  val items : opamfile_item list -> string
  val opamfile : opamfile -> string
end
  [@@ocaml.deprecated "Use OpamPrinter.FullPos.Normalise instead."]
```
The module deprecation has the benefit that it also displays on `open OpamPrinter.Normalise`